From: Andreas Henriksson <andreas@fatal.se>
Date: Sat, 20 Jul 2013 18:17:14 +0200
Subject: Fetch database credentials from external file

Modify php configuration to fetch postgresql auth params from
dbconfig-common generated file.
---
 phphtdocs/config.conf | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/phphtdocs/config.conf b/phphtdocs/config.conf
index 554db7a..a11c69e 100644
--- a/phphtdocs/config.conf
+++ b/phphtdocs/config.conf
@@ -1,4 +1,4 @@
-<?
+<?php
 define("DFLT_WIDTH", 1100);
 define("DFLT_HEIGHT", 325);
 define("DFLT_INTERVAL", INT_DAILY);
@@ -7,5 +7,6 @@ define("ZOOM_LONG", "-90.65995216369629");
 define("ZOOM_LAT", "40.72982797782921");
 define("ZOOM_LEVEL", "8");
 
-$db_connect_string = "host = 216.24.112.16 user = swarm dbname = swarm"
-?>
\ No newline at end of file
+include("/etc/bandwidthd/debian-db.php");
+$db_connect_string = "host='$dbserver' port='$dbport' dbname='$dbname' user='$dbuser' password='$dbpass'";
+?>
